dev-tools/omdb/tests/successes.out

```diff
@@ -401,14 +401,14 @@ task: "dns_propagation_external"
 
 
 task: "nat_v4_garbage_collector"
-  configured period: every 30s
+  configured period: every <REDACTED_DURATION>s
```

Comment:
Interesting, why did this change?

Reply:
We had it already for `ms`, so I figured `s` was an oversight?